Backpack/Rating/Classes/BPKRating.swift

```diff
@@ -208,6 +209,8 @@ public class BPKRating: UIView {
 
         updateSize()
         ratingValueLabel.text = ratingScale.displayedValue(from: value)
+
+        updateTitleSubtitleStackViewSubviews()
```

Comment:
I dont think you need to call this from here, the title and subtitle labels are added at setup time and are never removed